quickwit-metastore/src/metastore/index_metadata.rs

```diff
@@ -131,8 +131,8 @@ impl IndexMetadata {
         let search_settings = SearchSettings {
             default_search_fields: vec![
                 "body".to_string(),
-                "attributes.server".to_string(),
-                "attributes.server.status".to_string(),
+                r#"attributes.server"#.to_string(),
+                r#"attributes.server\.status"#.to_string(),
```

Comment:
not `r#"attributes\.server\.status"#.to_string()` ?

Reply:
no. the first "." is because of an object type. the second one is a quickwit field name with a "." in it.